---
title: "'Catch' の終わりには、対応する 'End Try' を指定しなければなりません"
ms.date: 07/20/2015
f1_keywords:
- bc30441
- vbc30441
helpviewer_keywords:
- BC30441
ms.assetid: 0e4756b4-1f29-4073-88c5-8f8c93ba6c9e
ms.openlocfilehash: 6955bc5e5d897cc5d1ccdc5e3bbf62296ccd6020
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 06/04/2020
ms.locfileid: "84361815"
---
# <a name="catch-must-end-with-a-matching-end-try"></a>'Catch' の終わりには、対応する 'End Try' を指定しなければなりません
`Catch` ステートメントが、一致する `End Try` ステートメントなしでコードに記述されています。 `Catch` ステートメントは、 `End Try` ステートメントで終了する必要があります。  
  
 **エラー ID:** BC30441  
  
## <a name="to-correct-this-error"></a>このエラーを解決するには  
  
1. `Catch` ステートメントを削除します。  
  
2. ブロックを終了するための `End Try` ステートメントを追加します。  
  
## <a name="see-also"></a>関連項目

- [Try...Catch...Finally ステートメント](../language-reference/statements/try-catch-finally-statement.md)
